### DISCUSSION TITLE
Can I compile a nextjs app

### DISCUSSION THREAD

**Discussion by jensysantana** [nexe/nexe#924] [Q&A] [answered]
Hi I would like to know if can I use nexe to compile a nextjs app? And how is the process?

**Comment by calebboyd** [comment, marked as answer]
Hello, Thanks for posting your question. Since this is not an Issue I'm going to convert it to a discussion. 

In order to get help from the users and contributors to this library, you may want to provide examples of what you've tried so far!

Thanks

**Comment by jensysantana** [comment]
So. I want to build a post system. With nextjs in my front end. So nextjs is use server side rendering with express under the hood. So if I can take my same post system and generate an executable will be good.

**Comment by jensysantana** [comment]
See this like that. If I can take my same nextjs code en generate an executable so at this point I no need the electron.

**Reply by somil24** [reply]
Hey were you able to find out a solution for the same ?

**Comment by wanghaisheng** [comment]
@calebboyd  for a project like this,is there any docs around build it to exe  

{
  "name": "big-agi",
  "version": "1.8.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "env:pull": "npx vercel env pull .env.development.local",
    "postinstall": "prisma generate",
    "db:push": "prisma db push",
    "db:studio": "prisma studio"
  },

**Comment by lordjoo** [comment]
Any updates on this
